lib/AST/Decl.cpp

```diff
@@ -2047,7 +2060,10 @@ void ValueDecl::setOverriddenDecls(ArrayRef<ValueDecl *> overridden) {
 
 bool ValueDecl::isObjC() const {
   ASTContext &ctx = getASTContext();
-  return ctx.evaluator(IsObjCRequest{const_cast<ValueDecl *>(this)});
+  auto result = ctx.evaluator(IsObjCRequest{const_cast<ValueDecl *>(this)});
+  if (result)
+    return result;
```

Comment:
That should return `*result`, right? I'm a bit nervous that this type-checked at all...

Reply:
It doesn’t type check 😅 still testing it locally and haven’t pushed. This is just a WIP so far